### PR TITLE
Remove namespace dependency and refactor cleaning up cluster.

### PR DESCRIPTION
**Pull request by mkibbe1993**

### CODE REVIEW

**Comment by duglin** [comment]
LGTM

File: script/cluster_utilities.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# limitations under the License.
+
+set -ux
+
+. "${ROOT}/script/utilities.sh" || { echo 'Cannot load Bash utilities'; exit 1; }
+
+# Cleanup services in Kubernetes to prevent network resource leaking
+function wipe_cluster() {
+  local namespace
+  for namespace in $(kubectl get namespaces -oname | grep -v kube-system); do
```
Comment (pmorie) [inline script/cluster_utilities.sh]: In the future, let's make this a little less scorched-earth and a little more tactical nuke.

File: script/cluster_utilities.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ux
+
+. "${ROOT}/script/utilities.sh" || { echo 'Cannot load Bash utilities'; exit 1; }
```
Comment (pmorie) [inline script/cluster_utilities.sh]: hack/utilities?

Also, this file needs to go in hack/

File: script/cluster_utilities.sh (hunk 1 above)
Comment (arschles) [inline script/cluster_utilities.sh]: Why do we have to clean up at all? I was under the impression that it was operating on a GKE cluster that would be torn down immediately after the e2e test run

Comment (martinmaly) [inline script/cluster_utilities.sh]: funny you should ask :) we tried it. Problem is if you just kill the cluster without wiping it clean first (and making sure the cleanup is done), GKE will leak resources which exist outside of the k8s cluster (most notably, network load balancers).

Comment (arschles) [inline script/cluster_utilities.sh]: so, are you deleting namespaces and then deleting the GKE cluster?

Comment (martinmaly) [inline script/cluster_utilities.sh]: in each namespace other than kube-system we delete everything (deployments, services, pods, ...) Then we delete the namespace.

Once done, we delete the cluster.

File: script/cluster_utilities.sh (hunk 2 above)
Comment (mkibbe1993) [inline script/cluster_utilities.sh]: Sorry about that, I was sloppy should have double checked things.

**Comment by arschles** [comment]
LGTM (`LGTM3` label added)

**Pull request merged**